jupyterhub/handlers/login.py

```diff
@@ -12,6 +12,9 @@
 class LogoutHandler(BaseHandler):
     """Log a user out by clearing their login cookie."""
     def get(self):
+        user = self.get_current_user()
+        if user:
+            self.log.info("User %s logged out", user.name)
```

Comment:
To make it easier to scan logs visually, I would prefer the username at the end:

    User logged out: username